spec/checkMergeConflictSpec.js

```diff
@@ -271,4 +290,174 @@ describe('Merge Conflict Check', () => {
       expect(github.issues.removeLabel).not.toHaveBeenCalled();
```

Comment:
Just wanted to confirm - what if the PR author is accidentally not assigned on a PR with a merge conflict label, do we assign them if we find this case?

Reply:
Initially, no. I've updated it to do that.